### PR TITLE
feat(examples): Add SavedModel

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Add savedmodel fashion_model as an example

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: examples/SavedModel-fashion/ormbfile.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+author: "Ce Gao <gaoce@caicloud.io>"
+format: SavedModel
```
Comment (ddysher) [inline examples/SavedModel-fashion/ormbfile.yaml:2]: Should we fill in more information here?

Comment (gaocegege) [reply examples/SavedModel-fashion/ormbfile.yaml:2]: Now I think it is enough. We can add more info soon. Because model config is not determined now.

Comment (ddysher) [reply examples/SavedModel-fashion/ormbfile.yaml:2]: LGTM

**Pull request merged**